ctapipe/image/toymodel.py

```diff
@@ -183,11 +188,9 @@ def from_camera_readout(cls, readout, gain_channel=0):
 
 
 class ImageModel(metaclass=ABCMeta):
-    @u.quantity_input(x=u.m, y=u.m)
```

Comment:
What is the background to all these Quantity wrappers being deleted?

Reply:
To support both telescope and camera frame, things need to support degrees and meters, unfortunately, `u.quantity_input` does not allow something like `x=(u.deg, u.m)` to say "either meters or degrees").

So instead now, we take the unit of `x` and check for consistency between the arguments by excplicitly calling `to_value(x.unit)` or comparing to the unit of the geometry